spec.bs

```diff
@@ -107,6 +107,16 @@ spec: CSP; urlPrefix: https://w3c.github.io/webappsec-csp/
 spec: CSPEE; urlPrefix: https://w3c.github.io/webappsec-cspee/
   type: dfn
     text: required csp; url: browsing-context-required-csp
+spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/
+  type: dfn
+    text: extension command; url: dfn-extension-commands

```

Comment:
[=extension command=] seems to be exported. This shouldn't be needed.
(my spec writing instruction doc listed 3 ways to find exported terms or to decide whether a term is exported)

Reply:
Removed.